arraykit.c

```diff
@@ -288,6 +289,22 @@ ArrayGO_extend(ArrayGOObject *self, PyObject *values)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+ArrayGO_getnewargs(ArrayGOObject *self, PyObject *Py_UNUSED(unused))
+{
+    if (self->list && update_array_cache(self)) {
+        return NULL;
+    }
+    PyObject *args = PyTuple_New(1); // gets ref count of 1, will pass on to caller
+    if (!args) {
+        return NULL;
+    }
+    Py_INCREF(self->array); // set_item steals a ref count, so inc before setting
+    PyTuple_SET_ITEM(args, 0, self->array);
+    return args;
+    // Py_INCREF(Py_None);
+    // return Py_None;
```

Comment:
Here's a cleaner way of writing everything you have here:

```suggestion
    return PyTuple_Pack(1, self->array);
```

Reply:
Oops, I missed this. Let me get it in there.